### PR TITLE
upstream changes

### PR DESCRIPTION
**Pull request by fivetran-jamie**
## PR Overview
**This PR will address the following Issue/Feature:**
https://github.com/fivetran/dbt_apple_store_source/issues/8

**This PR will result in the following new package version:**

v0.3.0  - upstream packages had breaking changes 

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**
the country code seed files changed so people need reseed

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt compile
- [x] dbt run –full-refresh
- [x] dbt run
- [x] dbt test
- [ ] dbt run –vars (if applicable)

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked and tagged
- [x] You are assigned to the corresponding issue and this PR
- [ ] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [x] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [x] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [x] You have provided details below around the validation steps performed to gain confidence in these changes.

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [x] indexed within dbt_project.yml
    - [x] indexed within integration_tests/dbt_project.yml
- [x] CHANGELOG has individual entries for each respective change in this PR
    
- [x] README updates have been applied (if applicable)
    
- [ ] DECISIONLOG updates have been updated (if applicable)
- [x] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [x] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
🧀

### CODE REVIEW

File: packages.yml
Comment (fivetran-joemarkiewicz) [inline packages.yml:11]: Reminder to update before merge

**Review by fivetran-joemarkiewicz — APPROVED** [review]
This looks good to go pending PR #15 (or the same adjustments) are applied to this PR.

File: README.md
Comment (fivetran-reneeli) [inline README.md:94]:
```suggestion
https://github.com/fivetran/dbt_apple_store_source/blob/main/seeds/apple_store_country_codes.csv
```

Comment (fivetran-reneeli) [reply README.md:94]: Think you meant to link the apple codes to this link, not google (404 error)

https://github.com/fivetran/dbt_apple_store_source/blob/main/seeds/apple_store_country_codes.csv

File: CHANGELOG.md
Comment (fivetran-reneeli) [inline CHANGELOG.md:6]:
```suggestion
  - This is a 🚨**breaking change**🚨 as you will need to re-seed (`dbt seed --full-refresh`) the `google_play__country_codes` [file](https://github.com/fivetran/dbt_google_play_source/blob/main/seeds/google_play__country_codes.csv) and `apple_store_country_codes` [file](https://github.com/fivetran/dbt_apple_store_source/blob/main/seeds/apple_store_country_codes.csv) again.
```

File: README.md
Comment (fivetran-reneeli) [inline README.md:93]:
```suggestion
You will need to `dbt seed` the `google_play__country_codes` [file](https://github.com/fivetran/dbt_google_play_source/blob/main/seeds/google_play__country_codes.csv) and `apple_store_country_codes` [file](https://github.com/fivetran/dbt_apple_store_source/blob/main/seeds/apple_store_country_codes.csv) just once.
```

File: README.md
Comment (fivetran-jamie) [reply README.md:94]: updated!

**Pull request merged**
